frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -593,9 +535,10 @@ function ReaderFooter:init()
         end
     end
 
+    self.face = Font:getFace(self.text_font_face, self.settings.text_font_size)
```

Comment:
I guess there's not much doubts what this "face" is used for (as footer does not display anything else), but it would read easier if named `self.item_face` or `self.footer_item_face`.